### PR TITLE
created model, and controller -- Matt double check

### PR DESCRIPTION
**Pull request by AlisonPAnderson**
I did copious googling!  Double check please!

### CODE REVIEW

File: backend/app/models/property_assessment.rb [hunk 1]
```diff
@@ -1,3 +1,64 @@
...
+            kitchen_style_1: kitchen_style1,
+            kitchen_style_2: kitchen_style2,
+            kitchen_style_3: kitchen_style3,
+            internal_condition: internal_condition,
+            external_condition: external_condition,
+            overall_condition: overall_condition,
+            bedrooms_count: bedrooms_count,
+            bathrooms_full_count: baths_full_count,
+            bathrooms_half_count: baths_half_count,
+            kitchens_count: kitchens_count
```
Comment (mzagaja) [inline backend/app/models/property_assessment.rb:59]:
```suggestion
            kitchens_count: kitchens_count,
```

File: backend/db/migrate/20220817011608_create_property_assessments.rb [hunk 2]
```diff
@@ -51,7 +53,7 @@ def change
       t.string :bthrm_style3
       t.string :kitchen_type
       t.string :kitchen_style1
-      t.string :kitchen_style
+      t.string :kitchen_style  # is this kitchen_style2?
```
Comment (mzagaja) [inline backend/db/migrate/20220817011608_create_property_assessments.rb:56]: I would nuke the comment from this PR and fix/update the field name in a different PR.

File: backend/Gemfile [hunk 3]
```diff
@@ -39,6 +39,7 @@ gem "bootsnap", require: false
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem "debug", platforms: %i[ mri mingw x64_mingw ]
+  gem 'rspec'
```
Comment (mzagaja) [inline backend/Gemfile:42]: We'll be using `rspec` but I would wait until we actually start writing tests to add the gem, there are also some other RSpec setup steps to do when you first introduce it.

**Review by mzagaja — CHANGES_REQUESTED** [review]
Made some suggestions to clean this up before we merge it in.

**Comment by AlisonPAnderson** [comment]
@mzagaja It was a whole adventure (learning experience! :) )to get my SSH set up on the linux shell so I realize I am a few days late on this commit and now there are conflicts.  My apologies! please advise!

**Pull request closed without merging**